src/components/Navbar/SettingsDrawer/SettingsBodySkeleton.tsx

```diff
@@ -0,0 +1,27 @@
+import Skeleton from '../../dls/Skeleton/Skeleton';
+
+import styles from './SettingsBodySkeleton.module.scss';
+
+const SETTINGS_ROW_COUNT = 4;
+const rowsArr = Array(SETTINGS_ROW_COUNT).fill(null);
```

Comment:
I feel this code is being duplicated in multiple places and It would be better to move it to a reusable Skeletons generator

Reply:
hmm, it is already two lines of code and only required in some of the skeletons based on the component, are you sure abstracting it is needed?

Reply:
So what I mean is not just this line but the the body of the `SettingsBodySkeleton` and other Skeletons too because I feel all of them share most of the code and what changes is just the className + the number of skeletons generated. But not very strong opinion